ports/libcaer/portfile.cmake

```
@@ -0,0 +1,25 @@
+vcpkg_from_gitlab(
+	GITLAB_URL https://gitlab.com/inivation/
+	OUT_SOURCE_PATH SOURCE_PATH
+	REPO dv/libcaer
+	REF 3.3.14
+	SHA512 6e91ebd20796b59c51ebb10be58d12577f3b6370425bbeffcf1a96ff91ad9f3ffaefb2741d0a932b241f2664c157d77158cf475b0f7e39ba208d5482f408fc8b
+	HEAD_REF ab9470e8900364822fb74ad3c1e99fa4088914df
+	PATCHES
+		libcaer-static-build.patch
+)
+
+vcpkg_cmake_configure(
+	SOURCE_PATH ${SOURCE_PATH}
+	OPTIONS
+		-DENABLE_OPENCV=ON
+		-DEXAMPLES_INSTALL=OFF
+		-DENABLE_BINDIR_INSTALLATION=ON
+)
+vcpkg_cmake_install()
+

```

```suggestion
vcpkg_fixup_pkgconfig
```
In order to avoid the location error generated by the `.pc` file, we can use this method to circumvent.
https://github.com/Microsoft/vcpkg/blob/master/docs/maintainers/vcpkg_fixup_pkgconfig.md :)

---

Thanks for your review comments, they are very helpful! I have added the function call you mentioned.